docs/reference/objects/company.md

```diff
@@ -4,60 +4,65 @@ title: Company
 parent: Objects
 ---
 
-Company is a _global_ variable, meaning it should always be accessible to you when developing on canvas
+Company is a _global_ variable, meaning it will always be accessible on any Liquid template.
 
-# company.products
-
-This will return an array of the companies published [products]({% link docs/reference/objects/product/index.md %}) excluding any private products
-e.g.
-
-{% raw %}
-```liquid
-{% for product in company.products %}
-    {{ product.name }}
-{% endfor %}
-```
-{% endraw %}
+# company.authors
 
+A list of all the company's [authors]({% link docs/reference/objects/author.md %}).
 
-# company.series
+# company.avatar_logo
 
-This will return an array of the companies [series]({% link docs/reference/objects/series.md %})
+The company's logo as an [image]({% link docs/reference/objects/image.md %}). The logo is set within **Settings** > **Your Brand**.
 
 # company.biographies
 
-This will return an array of the companies [biographies]({% link docs/reference/objects/biography.md %})
+A list of all the company's [biographies]({% link docs/reference/objects/biography.md %}).
 
 # company.countries
 
-This will return an array of the names of countries with products
-e.g.
+A list of the names of countries with products. e.g. `["Canada", "United Kingdom"]`
 
-# company.name
+# company.facebook_username
 
-Returns the name of the company
+The facebook username set within **My Site** > **Social**.
 
-# company.subdomain
+# company.instagram_username
 
-Returns the subdomain of the company
+The instagram username set within **My Site** > **Social**.
 
```

Comment:
move twitter here

Reply:
We've been going for method names to be alphabetical, rather than trying to group them in some other way. I know in certain cases, like this one, it feels counter-intuitive but overall I think it's worth it for the consistency. It will make method names easier to locate in the long-run.

Reply:
Ahh totally fair.